### PR TITLE
Adding GTM wideips - Branched out of Servers PR, merge this after PR(#673)

### LINKED ISSUE #633: add wideip support to GTM endpoint
this is required in order to implement gtm pools correctly

### PR DESCRIPTION
**Pull request by wojtek0806**
Fixes #633

Problem:
GTM WideIP  endpoint was missing from the SDK. This is a first of a series of PRs meant to introduce the GTM pools to the SDK

Analysis:
WideIP endpoint was added as it was a dependency for GTM pools

Tests:
Flake8
Functional Tests
Unit Tests

Files Added/Changed:

f5-common-python/f5/bigip/tm/gtm/init.py
f5-common-python/f5/bigip/tm/gtm/wideip.py
f5-common-python/f5/bigip/tm/gtm/test/test_wideip.py
f5-common-python/test/functional/tm/gtm/test_wideip.py

### CODE REVIEW

File: f5/bigip/tm/gtm/server.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        super(Virtual_Servers_s, self).__init__(server)
+        self._meta_data['allowed_lazy_attributes'] = [Virtual_Servers]
+        self._meta_data['required_json_kind'] = \
+            'tm:gtm:server:virtual-servers:virtual-serverscollectionstate'
+        self._meta_data['attribute_registry'] = {
+            'tm:gtm:server:virtual-servers:virtual-serversstate':
+                Virtual_Servers}
+
+
+class Virtual_Servers(Resource):
```
Comment (caphrim007) [inline f5/bigip/tm/gtm/server.py:68]: I realize I let this slide in the last PR, but we have a precedent in the sys/snmp code whereby the resource being operated on in singular. So methinks we should continue that here and fix the other PR (or fix both of these in a new PR).

See here
https://github.com/F5Networks/f5-common-python/blob/development/f5/bigip/tm/sys/snmp.py#L64

Comment (wojtek0806) [reply f5/bigip/tm/gtm/server.py:68]: huh ?

**Comment by caphrim007** [comment]
aside from the plural -> singular, lgtm. singular can be fixed for both this and #673 in a later pr

**Comment by wojtek0806** [comment]
ok can you open issue for that and assign to me :) ?

**Comment by jasonrahm** [comment]
I've seen it both ways..

this PR is in alignment with existing ltm/pool/members

**Pull request merged**